recipes/mutmut/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "mutmut" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d78bb3324a4da736c31c9fb19c197290ededaca76c8eac2c2e40e70b59f38720
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - mutmut = mutmut.__main__:climain
+
+# until https://github.com/conda-forge/pony-feedstock/pull/20 we need to constrain python <3.10
+requirements:
+  host:
+    - python >=3.7, <3.10
+    - pip
+  run:
+    - python >=3.7, <3.10
```

Comment:
With the referenced PR merged you shouldn't need the `<3.10` constraint, no?

Reply:
yes, you are perfectly right!

Reply:
looks like the CDN has not been updated yet to use the new build [`noarch/pony-0.7.14-pyhd8ed1ab_1.tar.bz2`](https://anaconda.org/conda-forge/pony/files)